mits/automated_emails.py

```diff
@@ -0,0 +1,20 @@
+from mits import *
+
+
+class MITSEmail(AutomatedEmail):
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('sender', c.MITS_EMAIL)
+        AutomatedEmail.__init__(self, MITSTeam, *args, **kwargs)
+
+MITSEmail('Thanks for showing an interest in MITS!', 'mits_registered.txt',
+          lambda team: not team.submitted and team.registered < datetime.now(UTC) - timedelta(hours=1),
```

Comment:
This is kind of an odd lambda... I'm not sure what it's meant to be doing. Same with the similar timedelta check below. Can you explain the reasoning behind making emails wait an hour?